stb_image_resize.h

```diff
@@ -959,7 +959,7 @@ static int stbir__edge_wrap_slow(stbir_edge edge, int n, int max)
     {
         if (n < 0)
         {
-            if (n < max)
```

Comment:
This condition is always true, because `n` is negative here, and `max` is always positive.
The overflow happens if `-n >= max`: the function still returns `-n` instead of `max - 1`.